components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -176,14 +179,19 @@ function CreditCardInputForm(props: { id: string }) {
             }
         } catch (error) {
             console.error(error);
-            alert(error?.message || "Failed to submit form. See console for error message.");
+            setBillingError(error?.message || "Failed to submit form. See console for error message.");
```

Comment:
I get that there might be some extra context in the the console that might be helpful, but I think it's pretty unusual for a production site to refer a user to the developer console for further information.

Would it be better to remove that sentence and add the extra information to the UI in a subsequent PR?

Reply:
Thanks for raising this! It's boilerplate code I kept from a Stripe snippet, but you're right about referring to the console. Instead, how about always stringifying the error? I think this works equally well with or without `error.message`:

```suggestion
            setBillingError(String(error));
```

Reply:
Yes, that looks like it will return the `.message` in the case where the error has one [[1](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toString)]

I think that's preferable to referring users to the console.